govcd/org_oidc.go

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2024 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+// GetOpenIdConnectSettings retrieves the current OpenID Connect settings for a given Organization
+func (adminOrg *AdminOrg) GetOpenIdConnectSettings() (*types.OrgOAuthSettings, error) {
+	result, err := oidcExecuteRequest(adminOrg, http.MethodGet, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+// SetOpenIdConnectSettings sets the OpenID Connect configuration for a given Organization. If the well-known configuration
+// endpoint is provided, the configuration is automatically retrieved from that URL.
+// If other fields have been set in the input structure, the corresponding values retrieved from the well-known endpoint are overridden.
+// If there are no fields informed, the configuration retrieved from the well-known configuration endpoint is applied as-is.
```

Comment:
What does "informed" mean in this context?

Reply:
I meant "set", changed.